cmd/config

```diff
@@ -0,0 +1 @@
+../config
```

Comment:
Why is there a symlink you might ask! The answer is we need the file in the tests and the executable and they are in different places and I didn't feel like writing a config for where to find the config file